### ISSUE TITLE
try to make "full-template" work

### ISSUE THREAD

**Issue by atherdon** [atherdon/markdown-to-email#385]
![image](https://user-images.githubusercontent.com/1469198/116010103-b8d68780-a625-11eb-8acc-67b9c96a7f37.png)

получается что мы начали работу над этими темплейтами, но она явно не очень была закончена.
предлагаю попробывать вывести какие то данные и посмотреть, как все работает в реакт ветке

сс @coder-do

**Comment by coder-do** [comment]
да full template не работает, вот почему - файл  https://github.com/atherdon/markdown-to-email/blob/react-components-stage-zero/templates/full/Body/BodyPrototype.js     пустой и поэтому кроме ошибок ничего не выводилось - я для теста взял контент этого же файла из `content-only` страницы и фулл тоже запустился. Я просто не знаю что должно быть в фулл темплейте(контент имею ввиду) если заполнить этот файл все сработает норм

**Comment by atherdon** [comment]
@coder-do да, я тоже уже успел покопаться в full темплейте. его текущая версия - это я уже нашаманил. но т.к. мне сложно нормально делать код - я начал пытаться вытащить папку misc в отдельный модуль. но пока что - не очень работает
